setup.py

```diff
@@ -57,9 +64,6 @@ def read(*rnames):
           'release': [
               'zest.releaser',
           ],
-          'test': [
-              'pytest',
-          ],
```

Comment:
This will break `RestrictedPython [test]` when used in buildout. (Do we need it?)
If I remember correctly buildout is not able to handle `test_requires`. So maybe we should have both using the same list of packages.